### PR TITLE
Fix alias command and prompt docker image on config init

### LINKED ISSUE #196: cl alias broken
cl alias throws an error currently.

Also, during the interactive setup, we should ask:
- Do you want to connect to a codalab server (in which case, we don't want to use MockAuthHandler)?
- Do you want to set up docker (in which case docker_image + q needs to be added)?

### PR DESCRIPTION
**Pull request by kashizui**
Fixes #196 
(happy 200th pull request 🎉)

@percyliang @pbhatnagar3

### CODE REVIEW

File: codalab/lib/codalab_manager.py [hunk 1]
```diff
@@ -209,6 +208,8 @@ def init_config(self, dry_run=False):
 online to set up execution with Docker:
 https://github.com/codalab/codalab/wiki/Dev_CodaLab%20CLI%20Execution%20in%20Docker
 """
+            if prompt_bool("Would you like to configure a Docker image to use?", default=False):
```
Comment (percyliang) [inline codalab/lib/codalab_manager.py]: Is the indentation off here?

**Comment by percyliang** [comment]
LGTM

**Comment by kashizui** [comment]
Added prompts for OAuth setup and made indentation clearer.

@percyliang

**Comment by percyliang** [comment]
LGTM again - thanks!

**Pull request merged**